tools/font-edit/twin-fedit.c

```diff
@@ -608,23 +645,29 @@ static void write_char(char_t *c)
     offset += 1;
 }
 
-int main(int argc, char **argv)
+static void generate_font_metrics(void)
 {
-    char_t *c;
     int ucs4;
-
-    if (!init(argc, argv))
-        exit(1);
-    while ((c = read_char())) {
-        play(c);
-        write_char(c);
-    }
     for (ucs4 = 0; ucs4 < 0x80; ucs4++) {
         if ((ucs4 & 7) == 0)
             printf("\n   ");
         printf(" %4d,", offsets[ucs4]);
     }
     printf("\n");
+}
+
+int main(int argc, char **argv)
+{
+    char_t *c;
+
+    if (!init(argc, argv))
```

Comment:
Dump the usage before calling `exit`.

Reply:
```diff
+    char_t *c;
+
+   /*
+    * Use the following command to start twin-fedit:
```
> Don't do that. Dump the message.

What kind of usage should I dump here?

Reply:
You can simply change the behavior of this font editor, allowing read a file from given filename (`argv[1]`). Thus, you can check argument count and the existence of file to proceed.